forum/templates/users/password_reset.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Password Reset</title>
+</head>
+<body>
+    <h2>Відновлення пароля</h2>
```

Comment:
Just recomendation(If not nessesary then just skip this comment)

Use main tag for better html structure, wrap like:
<body>
    <main>
        <-- content -->
    </main>
</body>